projects/Allwinner/devices/H6/patches/u-boot/002-orange-pi-3-support.patch

```diff
@@ -1,363 +0,0 @@
-diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
```

Comment:
Why did you delete this file? It's not in v2019.7, so I guess it needs just slight update.

Reply:
Doesn't apply and forget to fix it before submiting the PR